scamp/scamp-3.c

```diff
@@ -1244,9 +1245,9 @@ void proc_100hz (uint a1, uint a2)
                 }
             }
         }
-      else if (netinit_tick_counter >= 3)
+      else if (netinit_biff_tick_counter >= 20)
```

Comment:
> I missed this before my comment. I hadn't been getting any issues, but wondered if this improved things.

No worries! I'd be keen to revert simply because future readers may wonder if it really does need to be >3 to function correctly and then have to start reading a lot of code to try and work out what is going on.